src/main/java/com/example/security/TokenAuthenticationService.java

```diff
@@ -28,7 +28,7 @@ static void addAuthentication(HttpServletResponse res, String username) {
     }
 
     static Authentication getAuthentication(HttpServletRequest request) {
-        String token = request.getHeader(HEADER_STRING);
+        String token = request.getHeader(HEADER_STRING).substring(7); // skip the 'Baerer '
```

Comment:
It was not right that either on github or article because it was not there basically.